langchain/src/llms/openai-chat.ts

```diff
@@ -326,11 +326,13 @@ export class OpenAIChat extends LLM implements OpenAIInput {
       });
       this.client = new OpenAIApi(clientConfig);
     }
-    return this.caller.call(
-      this.client.createChatCompletion.bind(this.client),
-      request,
-      options
-    );
+    return this.caller
+      .call(
+        this.client.createChatCompletion.bind(this.client),
+        request,
+        options
+      )
+      .then((res) => res.data);
```

Comment:
why are you removing `.then((res) => res.data)` in some places but still using it here?

not saying it's wrong; just making sure cause it looks inconsistent

Reply:
It's being moved here (from where this function is called)